src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -1,5 +1,5 @@
 import { EmulatorLog } from "./types";
-import { CloudFunction, DeploymentOptions } from "firebase-functions";
+import { CloudFunction, DeploymentOptions, https } from "firebase-functions";
```

Comment:
@abeisgoat any issue with this import?

Reply:
What issue do you think this may cause?

Reply:
I don't think anything, I just wanted to double-check as I know deps in this file are super sensitive.